packages/donate-button-beta/src/components/widget/CurrencySuggestion/index.tsx

```diff
@@ -61,7 +61,8 @@ export const CurrencySuggestion = forwardRef(
 
 		useEffect(() => {
 			setShowSuggestion(
-				!currency.countriesCode?.includes(country?.countryCode)
+				Boolean(country?.countryCode) &&
+					!currency.countriesCode?.includes(country?.countryCode)
```

Comment:
I think typescript could narrow more cleanly like so:

```suggestion
				country?.countryCode
					? !currency.countriesCode?.includes(country.countryCode)
					: false
```

Reply:
also i think `countryCodes` instead of `countriesCode` is more sensical lol